src/sid/contacts.py

```diff
@@ -418,16 +433,24 @@ def _calculate_infections_by_random_contacts(
                     random_contacts[j, cm] -= 1
 
                     if infectious[i] and not immune[j]:
-                        infection_counter[i] += 1
-                        infected[j] = 1
-                        immune[j] = True
-                        was_infected_by[j] = cm
+                        is_infection = boolean_choice(
```

Comment:
According to the paper on which we base our numbers, all people older than 20 will have a multiplier of 1. I would expect a speed up if we check for this, either here or inside the boolean choice function.

Reply:
Given the following benchmark, I would leave it as it is.

![image](https://user-images.githubusercontent.com/22533006/108102820-5db67080-7089-11eb-886d-dd08da4bd672.png)